src/Blueprint/DatabaseBlueprint.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Reliese\Blueprint;
+
+use Reliese\Meta\DatabaseInterface;
+use RuntimeException;
+
+/**
+ * Class DatabaseManager
+ */
+class DatabaseBlueprint
+{
+    /**
+     * The name of the connection from the Laravel config/database.php file
+     *
+     * @var string
+     */
+    private $connectionName;
+
+    /**
+     * @var \Illuminate\Database\Connection
+     */
+    private $connection;
+
+    /**
+     * @var SchemaBlueprint[]
+     */
+    private $schemaBlueprints;
```

Comment:
Even though PHP might work this way, we should give an initial value to our variables.
```suggestion
    private $schemaBlueprints = [];
```